Source/WebCore/platform/cocoa/DragDataCocoa.mm

```diff
@@ -159,6 +159,19 @@ static inline String tiffPasteboardType()
 {
 }
 
+DragData::DragData(const String& dragStorageName, const IntPoint& clientPosition, const IntPoint& globalPosition, const Vector<String>& fileNames, OptionSet<DragOperation> sourceOperationMask, OptionSet<DragApplicationFlags> flags, OptionSet<DragDestinationAction> destinationActionMask, std::optional<PageIdentifier> pageID)
+    : m_clientPosition(clientPosition)
+    , m_globalPosition(globalPosition)
+    , m_platformDragData(0)
```

Comment:
Can this be nullptr?
Can this be an initializer in the header instead of here?

Reply:
I copied this constructor from the instance directly above which initialised m_platformDragData in the same way.

Do you mean, can we switch the declaration of m_platformDragData to default initialise to NULL and then exclude it from the initialiser list?

DragDataRef m_platformDragData { NULL };